src/applications/mhv/medical-records/containers/App.jsx

```diff
@@ -184,7 +184,7 @@ const App = ({ children }) => {
         <MrBreadcrumbs />
         <DowntimeNotification
           appTitle="Medical Records"
-          dependencies={[externalServices.mhv]}
+          dependencies={[externalServices.mhvPlatform, externalServices.mhvMr]}
```

Comment:
Just confirming if we want to depend on both services? Is it possible that the mhv platform service could be down without affecting already logged in users using the apps?

Reply:
The idea with mhvPlatform is that if it is set, then MR, SM and Meds will all show a Downtime Banner.  I did not rename the original `mhv` service in Pager Duty that is tied to the MHV Auth.

Reply:
oh of course, I forgot that you made a new service for mhv platform. ignore me, keep calm and carry on 😁